From: Jan Engelhardt <jengelh@inai.de>
Date: Sat, 13 Sep 2014 17:12:43 +0200
Subject: first_run: deactivate autoupdate globally

We currently do not have any update service infrastructure in place
(i.e. on our webserver), and sending update requests to Google also
sounds meh (wrong provider, after all).

Also, there is no tunable in chrome://settings (or similar) yet to
turn it back on.

License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
---
 chrome/browser/extensions/extension_system_impl.cc | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/chrome/browser/extensions/extension_system_impl.cc b/chrome/browser/extensions/extension_system_impl.cc
--- a/chrome/browser/extensions/extension_system_impl.cc
+++ b/chrome/browser/extensions/extension_system_impl.cc
@@ -204,8 +204,7 @@ void ExtensionSystemImpl::Shared::Init(bool extensions_enabled) {
 
   user_script_manager_ = std::make_unique<UserScriptManager>(profile_);
 
-  bool autoupdate_enabled = !profile_->IsGuestSession() &&
-                            !profile_->IsSystemProfile();
+  bool autoupdate_enabled = false;
 #if BUILDFLAG(IS_CHROMEOS_ASH)
   if (!extensions_enabled ||
       ash::ProfileHelper::IsLockScreenAppProfile(profile_)) {
--
2.25.1
